CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to Quicksilver
+
+Thank you for taking the time to contribute to Quicksilver! Your support really means a lot.
+
+There are two main ways to contribute: [opening issues](#issues) and [submitting patches](#prs).
+
+## Issues
+
+When you're opening an issue, generally there are three main categories.
+
+The first is a question: how do I build this, why is this structure named this, or what does this error message mean.
+Please make sure to describe your environment if you're having a problem you need help with!
+Including your operating system and its version, your rust compiler version (`rustc -vV`), and any commands you may have run, as well as their output.
+
+The second category of issue is a bug report.
+Make sure to describe your environment as above, and to include a snippet of code that demonstrates the bug.
+Without a way to reproduce the bug, it will be very hard to fix, so make sure to provide as much information as possible.
+
+The third category of issue is a feature request.
+Quicksilver is an evolving project that can definitely use a lot more features, and the project is very open to suggestions.
+However, there is a certain category of feature that (generally) won't be considered for work: anything that will only work on one of desktop or web.
+A firm goal of quicksilver is unsurprising behavior when taking a web-first project and compiling for desktop and vice-versa.
+Also, there's no guarantee anyone else will be up to working on your feature request; if that's the case, then feel free to [open a PR](#prs).
+
+
+## PRs
+
+The first rule of PRs to Quicksilver to generally try to make only a few changes in a single PR.
```

Comment:
This line might be missing an ``is`` in "... Quicksilver ``is`` to generally ...".
If preferred, the entire line could be shortened to:
"Generally, try to make only a few changes in a single PR to Quicksilver."

Reply:
Thanks, fixed